src/__tests__/changelog/getPackageChangelogs.test.ts

```diff
@@ -0,0 +1,71 @@
+import { getPackageChangelogs } from '../../changelog/getPackageChangelogs';
+import { BumpInfo } from '../../types/BumpInfo';
+import { ChangeSet } from '../../types/ChangeInfo';
+import { PackageInfos } from '../../types/PackageInfo';
+
+describe('getPackageChangelogs', () => {
+  it('should have multiple comment entries when a package has a changefile AND was part of a dependent bump', () => {
+    const changeFileChangeInfos: ChangeSet = new Map([
+      [
+        'foo.json',
+        {
+          comment: 'comment for foo',
+          commit: 'deadbeef',
+          dependentChangeType: 'patch',
+          email: 'something@something.com',
+          packageName: 'foo',
+          type: 'patch',
+        },
+      ],
+      [
+        'bar.json',
+        {
+          comment: 'comment for foo',
```

Comment:
Should be `comment for bar`? Just wanted to point this out in case it's causing any false positives. Seems there are more below too

Reply:
Arrrgh, typo! But the results are the same - test is verifying counts of things based on keys not value mostly.